inc/class-cachify-cli.php

```diff
@@ -55,53 +55,50 @@ public static function get_cache_size( $args, $assoc_args ) {
 
 		WP_CLI::line( $message );
 	}
+}
 
-	/**
-	 * Register CLI Commands
-	 *
-	 * @since   2.3.0
-	 * @change  2.3.0
-	 */
-	public static function add_commands() {
-		// Add flush command.
-		WP_CLI::add_command(
-			'cachify flush',
+/**
+ * Register CLI Commands
+ *
+ * @since   2.3.0
+ * @change  2.3.0
+ */
+/* Add flush command */
+WP_CLI::add_command(
```

Comment:
This is a class file which should preferably define a class. With this change, the file has 2 purposes, define the class and execute code. I personally dislike that mixture, if there is no significant benefit.

Is there a point I don't see right now, why this way of calling should be preferred over `add_action( 'cli_init', array( 'Cachify_CLI', 'add_commands' ) );` ?